src/main/java/edu/ksu/canvas/attendance/controller/AttendanceBaseController.java

```diff
@@ -20,6 +20,7 @@
 import org.springframework.web.servlet.ModelAndView;
 
 import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.servlet.http.HttpSession'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
You missed an import @jzheng21 Intellij has an option to optimize imports on the fly that's helpful for not needing to worry about removing unused imports. 
![optimizeimports](https://user-images.githubusercontent.com/4078791/41130020-6ab9b674-6a7a-11e8-850d-47daccd08424.png)